inspirehep/modules/records/receivers.py

```diff
@@ -103,6 +103,30 @@ def assign_uuid(sender, record, *args, **kwargs):
 # models_committed
 #
 
+@models_committed.connect
+def notify_legacy(sender, changes):
+    """Notify Legacy that a given record has been inserted or updated.
+
+    Notifies Legacy that a given record has been inserted or updated, by
+    sharing this information via Redis. Then Legacy will fetch the actual
+    record through subsequent HTTP requests.
+    """
+    redis_connection = None
+
+    def _get_redis_connection():
+        if redis_connection:
+            return redis_connection
+        redis_url = current_app.config.get('CACHE_REDIS_URL')
+        return StrictRedis.from_url(redis_url)
```

Comment:
This is a cheap way to cache the redis connection. If you have a better one, please do!